yt_dlp/extractor/radioradicale.py

```diff
@@ -0,0 +1,64 @@
+from locale import LC_ALL, setlocale
+from .common import InfoExtractor
+from urllib.parse import urlparse
+from re import sub
+from datetime import datetime
+from os.path import dirname
+
+
+class RadioRadicaleIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?radioradicale\.it/scheda/(?P<id>[0-9]+)'
+    _TESTS = [{
+        'url': 'https://www.radioradicale.it/scheda/471591',
+        'info_dict': {
+            'id': '471591',
+            'ext': 'mp4',
+            'title': 'md5:e8fbb8de57011a3255db0beca69af73d',
+            'creator': 'Giuseppe Di Leo',
+            'location': 'Napoli',
+            'timestamp': 1460044800.0,
+            'upload_date': '20160407',
+            'description': 'md5:5e15a789a2fe4d67da8d1366996e89ef',
+            'thumbnail': 'https://www.radioradicale.it/photo400/0/0/9/0/1/00901768.jpg',
+        }
+    }]
+
+    def _real_extract(self, url):
+        setlocale(LC_ALL, '')
+
+        video_id = self._match_id(url)
+        webpage = self._download_webpage(url, video_id)
+
+        video_info = self._parse_json(self._search_regex(
+            r'jQuery\.extend\(Drupal\.settings\s*,\s*({.+?})\);',
+            webpage, 'drupal settings'), video_id)['RRscheda']
+
+        json_ld = list(self._yield_json_ld(webpage, video_id))[0]
```

Comment:
Why not `_search_json_ld` directly?

Reply:
this one is one that I can answer.

`_search_json_ld` parses and cleans the json, `_yield_json_ld` only searches it without modification. I do not know why but _search was cleaning the json in the webpage too much removing extra infos I was looking for.